README.md

```diff
@@ -85,7 +85,8 @@ Travis uses a .travis.yml file in the root of your repository to learn about you
 | env            |Allows you to add new travis job matrix entries based on the included environmnet variables, one per `env` entry; for example, for adding jobs with specific `PUPPET_GEM_VERSION` and/or `CHECK` values.  See the [Travis Environment Variables](https://docs.travis-ci.com/user/environment-variables) documentation for details.|
 | global\_env     |Allows you to set global environment variables which will be defined for all travis jobs; for example, `PARALLEL_TEST_PROCESSORS` or `TIMEOUT`.  See the [Travis Global Environment Variables](https://docs.travis-ci.com/user/environment-variables/#Global-Variables) documentation for details.|
 |docker\_sets     |Allows you to configure sets of docker to run your tests on. For example, if I wanted to run on a docker instance of Ubuntu I would add  `set:docker/ubuntu-14.04` to my docker\_sets attribute.  The docker_sets is a hash that supports the 'set', 'testmode', and 'collection' keys. |
-|docker\_sets['set']| This should refrence the docker nodeset that you wish to run. |
+|docker\_sets['set']| This should reference the docker nodeset that you wish to run. |
+|docker\_sets['setfile']| This should reference the docker nodeset generated by beaker-hostgenerator that you wish to run. |
```

Comment:
nit: This should probable be

`This should reference the docker nodeset file generated by beaker-hostgenerator that you wish to run.`  Added the word file.